apps/mainsite/seeds/06_badgeinstances.py

```diff
@@ -1,23 +1,23 @@
 from cmath import exp
-from venv import logger
 from datetime import timedelta
+from venv import logger
+
+from badgeuser.models import BadgeUser, StudentAffiliation
 from django.conf import settings
+from django.db.models import QuerySet
 from django.utils import timezone
-
-from badgeuser.models import BadgeUser
+from institution.models import Faculty, Institution
 from issuer.models import BadgeClass, BadgeInstance, Issuer
-from institution.models import Institution, Faculty
 from lti_edu.models import StudentsEnrolled
 from mainsite.models import BadgrApp
-from django.db.models import QuerySet
 
 super_user = BadgeUser.objects.get(username=getattr(settings, 'SUPERUSER_NAME'))
 
 # By default, badge instances expire in 6 months, we will override this per badge instance so several are expired
 months_ahead = 6
 
 
-def all_students() -> QuerySet[BadgeUser]:
+def all_students_in(institution: Institution) -> QuerySet[BadgeUser]:
     return BadgeUser.objects.filter(
```

Comment:
Don't you miss the filter for institution here?

Reply:
Good catch, and added in 21cfc14